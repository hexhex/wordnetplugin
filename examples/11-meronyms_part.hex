%% this returns all (part) meronyms of a word
%% POS: nouns

%% the search number for (part) meronyms
%% is 11.
%% Note: In the WordNet documentation this is
%%       the search number for part holonyms.
%%       In fact search 11 fetches words, for
%%       which *word* is a holonym (and these
%%       are its meronyms)

word("car").
meronyms(W,SW,M,SM) :- &wordnet[11,W](1,SW,M,SM),word(W).
