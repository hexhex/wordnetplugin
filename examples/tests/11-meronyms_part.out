{meronyms("car",1,"window",29), meronyms("car",1,"third",28), meronyms("car",1,"third_gear",28), meronyms("car",1,"fin",27), meronyms("car",1,"tailfin",27), meronyms("car",1,"tail_fin",27), meronyms("car",1,"sunshine-roof",26), meronyms("car",1,"sunroof",26), meronyms("car",1,"anti-sway_bar",25), meronyms("car",1,"stabilizer_bar",25), meronyms("car",1,"running_board",24), meronyms("car",1,"roof",23), meronyms("car",1,"reverse_gear",22), meronyms("car",1,"reverse",22), meronyms("car",1,"rear_window",21), meronyms("car",1,"trunk",20), meronyms("car",1,"automobile_trunk",20), meronyms("car",1,"luggage_compartment",20), meronyms("car",1,"cowling",19), meronyms("car",1,"cowl",19), meronyms("car",1,"bonnet",19), meronyms("car",1,"hood",19), meronyms("car",1,"high",18), meronyms("car",1,"high_gear",18), meronyms("car",1,"radiator_grille",17), meronyms("car",1,"grille",17), meronyms("car",1,"glove_compartment",16), meronyms("car",1,"petrol_engine",15), meronyms("car",1,"gasoline_engine",15), meronyms("car",1,"floorboard",14), meronyms("car",1,"low",13), meronyms("car",1,"low_gear",13), meronyms("car",1,"first",13), meronyms("car",1,"first_gear",13), meronyms("car",1,"wing",12), meronyms("car",1,"car_window",11), meronyms("car",1,"car_seat",10), meronyms("car",1,"car_mirror",9), meronyms("car",1,"car_door",8), meronyms("car",1,"bumper",7), meronyms("car",1,"fender",6), meronyms("car",1,"fender",12), meronyms("car",1,"buffer",6), meronyms("car",1,"hooter",5), meronyms("car",1,"horn",5), meronyms("car",1,"motor_horn",5), meronyms("car",1,"car_horn",5), meronyms("car",1,"automobile_horn",5), meronyms("car",1,"automobile_engine",4), meronyms("car",1,"auto_accessory",3), meronyms("car",1,"air_bag",2), meronyms("car",1,"gun",1), meronyms("car",1,"throttle",1), meronyms("car",1,"gas",1), meronyms("car",1,"gas_pedal",1), meronyms("car",1,"accelerator_pedal",1), meronyms("car",1,"accelerator",1), meronyms("car",2,"suspension_system",1), meronyms("car",2,"suspension",1), word("car")}
